doc/source/ray-core/actors.rst

```diff
@@ -380,6 +380,47 @@ If we instantiate an actor, we can pass the handle around to various tasks.
             }
 
 
+Cancelling  actor tasks
+-----------------------
+
+Actor tasks can be canceled by calling :func:`ray.cancel() <ray.cancel>` on the returned Object ref.
+
+.. tab-set::
+
+    .. tab-item:: Python
+
+        .. literalinclude:: doc_code/actors.py
+            :language: python
+            :start-after: __cancel_start__
+            :end-before: __cancel_end__
+
+
+In Ray, task cancellation behavior is contingent on the task's current state:
+
+**Unscheduled Tasks**:
+If the actor task hasn't been scheduled yet, Ray attempts to cancel the scheduling. 
+When successfully cancelled at this stage, invoking ``ray.get(actor_task_ref)`` 
+will produce a :class:`TaskCancelledError <ray.exceptions.TaskCancelledError>`.
+
+**Running Actor Tasks (Regular Actor, Threaded Actor)**:
+For tasks classified as single threaded actor or a multi-threaded actor,
+Ray offers no mechanism for interruption.
```

Comment:
future work: we should really just add a `ray.is_cancelled()` method that folks can poll (I think you mentioned it in the past)

Reply:
Let me create an issue

Reply:
https://github.com/ray-project/ray/issues/39152